docs/gpa.md

```diff
@@ -14,6 +14,28 @@ This document annotates selected parts of the GPA, with details on how to publis
 
    References from Articles VII:4, VII:5, IX:8, IX:9, X:7, X:9, X:11, XIII:2, XIV:1, XVI:2 are not yet added.
 ```
+<div markdown=1>
+
```

Comment:
I don't think we need this div. We only need `markdown=1` if the Markdown content is inside an HTML element (like a `<table>`).

Reply:
Done